traffic-control-openshift/README.md

```diff
@@ -0,0 +1,62 @@
+# Traffic Control on OpenShift
+
+## What do you need to start?
+- Get a OpenShift ready VM [here](https://github.com/kinvolk/openshift-evangelists-vagrant-origin) to run the demo. You need to __reboot__ the VM after you build it.
+- Get get the demo repository [here](https://github.com/kinvolk/demo)
+- Copy the .yaml files into the VM
+
+By using ``` vagrant up ``` you run the VM, with ``` vagrant ssh ``` you can login, and ``` vagrant halt``` stops it.
+
+You can reach the OpenShift dashboard with your browser at [https://10.2.2.2:8443](https://10.2.2.2:8443). Use user 'admin' and password 'admin' to login, now you are in the OpenShift dashboard.
+
+For convenience you should add this to ~/.ssh/config
```

Comment:
Since the user will have to fix the config ("/path/to/your/demo/directory/"), you could instead explain how to get the base config (`vagrant ssh-config`), and only mention the lines they need to add (the `LocalForward`).